package.json

```
@@ -30,8 +31,12 @@
     "type-fest": "^2.19.0",
     "typescript": "^5.2.2"
   },
+  "optionalDependencies": {
+    "corepack": "0.20.0"
```

_(could)_ slightly loosen range to not have to manually apply any future non-breaking changes/fixes:

```suggestion
    "corepack": "~0.20.0"
```

---

There's nothing to upgrade to.  `corepack` is on v0.22.0 now, and requires Node.js v18.